### ISSUE TITLE
Use GitHub Actions Rather than Travis

### ISSUE THREAD

**Issue by grant** [GoogleCloudPlatform/functions-framework-dotnet#115]
For various reasons, we should use GitHub actions for CI rather than Travis:

## Expected

- Use GitHub Actions
- No Travis config: https://github.com/GoogleCloudPlatform/functions-framework-dotnet/blob/master/.travis.yml

---

Here's an example GitHub Action workflow we can use:
https://github.com/actions/setup-dotnet

```yaml
steps:
- uses: actions/checkout@main
- uses: actions/setup-dotnet@v1
  with:
    dotnet-version: '3.1.x' # SDK Version to use; x will use the latest version of the 3.1 channel
- run: ./build.sh
```

**Comment by jskeet** [comment]
(Discussing internally.)

**Comment by jskeet** [comment]
Fixed in #116.
